### PR TITLE
AYR-962/ update legacy code of sqlalchemy 

### PR DESCRIPTION
**Pull request by pgandhizaizi**
## Changes in this PR
1. updated legacy query.get references for sqlalchemy in routes.py

## JIRA ticket

AYR - 962

### After

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by pgandhizaizi** [comment]
executed all unit test cases
image

**Comment by pgandhizaizi** [comment]
executed e2e tests for all browser (visual regression for chromium failing due to pixel issues)
image

**Comment by colinbowen** [comment]
There looks to be an updated query that we can use for file = File.query.get_or_404(record_id) inside of routes.py similar to what you've done with the other queries e.g. file = db.session.get(File, record_id).

Will then have a check to just return a 404 if not found after.

File: app/main/routes.py
Comment (colinbowen) [inline app/main/routes.py:1]: There looks to be one query left using legacy SQLAlchemy:
`file = File.query.get_or_404(record_id)`

Comment (colinbowen) [reply app/main/routes.py:1]:
```
file = db.session.get(File, record_id)

if file is None:
    abort(404)
```

Comment (pgandhizaizi) [reply app/main/routes.py:1]: done the changes as requested and removed the use of get_or_404() function to remove sqlalchemy warning

**Review by colinbowen — APPROVED** [review]

**Pull request merged**